client/components/chart/legend.js

```diff
@@ -14,7 +15,7 @@ import { formatCurrency } from '@woocommerce/currency';
 /**
  * Internal dependencies
  */
-import './style.scss';
+import './legend.scss';
```

Comment:
Why the change to sass file name?

Reply:
`style.scss` was getting quite big so I decided to divide it in two files.

`style.scss` is still called from `charts.js` so it's not a problem removing it here.